### DISCUSSION TITLE
Read dataloader

### DISCUSSION THREAD

**Discussion by jihyeheo** [Project-MONAI/MONAI#6052] [Q&A]
It's an error that occurs when loading data, but I don't know

Error
Please install the reader libraries, see also the installation instructions:
https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies.
The current registered: [<monai.data.image_reader.NumpyReader object at 0x7fcb31abd9d0>, <monai.data.image_reader.PILReader object at 0x7fcb31abdf10>].

What should I do?

**Comment by KumoLiu** [comment]
Hi @jihyeheo, could you please try `pip show nibabel`? Seems you didn't have a `NibabelReader`.

**Reply by jihyeheo** [reply]
Thank you for your reply. 



There was no "nibabel", so when I played it again after installation, 

The current registered: [<monai.data.image_reader.NumpyReader object at 0x7f11e738bf90>, <monai.data.image_reader.PILReader object at 0x7f11e7398050>, <monai.data.image_reader.NibabelReader object at 0x7f11e7398090>].

increased by one. what can i install?

**Reply by KumoLiu** [reply]
Hi @jihyeheo, so after installation, you still can't load data, right?

**Reply by KumoLiu** [reply]
May I ask what data you are trying to load?

**Reply by jihyeheo** [reply]
Hi, @KumoLiu It is raw image data. Shouldn't it be possible to put the image data path into the dataloader?

**Reply by KumoLiu** [reply]
Not sure what you mean. You could find some references in the [tutorial repo](https://github.com/Project-MONAI/tutorials). Or maybe share a piece of your code, then I could take a deep look at it. Thanks!

**Reply by jihyeheo** [reply]
Sorry, I don't speak english ..... The .raw file failed to enter data into the loader?

**Reply by wyli** [reply]
the data loading tutorial shows some typical use cases: https://github.com/Project-MONAI/tutorials/blob/main/modules/load_medical_images.ipynb
